Symfony/CS/Tests/Fixer/Contrib/AliasFunctionsFixerTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ *
+ * Fixes AliasFunctionsUsageInspection warnings from Php Inspections (EA Extended)
+ */
+class AliasFunctionsFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @var array|string[]
```

Comment:
@kalessil 
`Tokens::getImportUseIndexes` returns array or array of arrays. Here you return array of strings, no alternative.